aspnetcore/blazor/components.md

```diff
@@ -511,6 +511,129 @@ Use `EventCallback` and `EventCallback<T>` for event handling and binding compon
 
 Prefer the strongly typed `EventCallback<T>` over `EventCallback`. `EventCallback<T>` provides better error feedback to users of the component. Similar to other UI event handlers, specifying the event parameter is optional. Use `EventCallback` when there's no value passed to the callback.
 
+## Chained bind
+
+A common scenario when creating nested components is the desire to bind a property:
+
+* To a page element in a component.
+* Across nested components.
```

Comment:
I'm not really sure what "nested components" means here.

Reply:
Components that use other components ... i.e., a graph of two or more components.

This is trying to say that a component's property is bound to one of its elements and to a field/property of a parent component.

Reply:
How about something like this?: "A common scenario when authoring components is to support chaining a data bound parameter to a page element in the component's output."